packages/server/src/api/routes/tests/row.spec.ts

```diff
@@ -685,4 +685,47 @@ describe("/rows", () => {
       expect(row._id).toEqual(existing._id)
     })
   })
+
+  describe("view search", () => {
+    function priceTable(): Table {
+      return {
+        name: "table",
+        type: "table",
+        schema: {
+          Price: {
+            type: FieldType.NUMBER,
+            name: "Price",
+            constraints: {},
+          },
+          Category: {
+            type: FieldType.STRING,
+            name: "Category",
+            constraints: {
+              type: "string",
+            },
+          },
+        },
+      }
+    }
+
+    it("returns table rows from view", async () => {
+      const table = await config.createTable(priceTable())
+      const rows = []
+      for (let i = 0; i < 10; i++)
+        rows.push(await config.createRow({ tableId: table._id }))
+
+      const createViewResponse = await config.api.viewV2.create()
+
+      const response = await request
```

Comment:
Would be good to get this request into the new views test client

Reply:
In this case, we are testing this endpoint, for this I use it explicitly instead of using the helpers. Don't you think it's better than using the helpers for it?

Reply:
Changed as discussed